week-2/L-array-methods-2/exercise3.js

```diff
@@ -7,15 +7,15 @@
 var ukNations = ["Scotland", "Wales", "England", "Northern Ireland"];
 
 function isInUK(country) {
-  return; // complete this statement
+  if (ukNations.includes(country)) return true;
+  else return false; // complete this statement
```

Comment:
This is correct but there is some unnecessary code here.

What is the value of `ukNations.includes(country)`? Is it a boolean? If it is, do need to write out `return true` or ` return false`? Or can we just return the boolean?